profile/README.md

```diff
@@ -0,0 +1,65 @@
+<p align="center">
+  <img src="https://security-summer-school.github.io/SSS-logo.png" width="215" alt="Security Summer School" />
+</p>
+<p align="center">
+  <img src="https://img.shields.io/github/contributors/security-summer-school/binary" />
+    <a href="https://bit.ly/DiscordSecuritySummerSchool">
+    <img src="https://img.shields.io/discord/715162845305634816.svg?label=discord&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" />
+  </a>
+</p>
+
+### Security Summer School is a security workshop held by people passionate about security, professors, PhD, MSci, BSc students from University POLITEHNICA of Bucharest and industry partners Ixia, a Keysight Business.
```

Comment:
- This message is too long for a `###` tag. Try simple bold or nothing at all
- Add GRX as a partner so @raresvis doesn't get sad

Reply:
Also PenTest tools as partner.

Reply:
Done.